src/class-fields/default/cls-decl-extends.template

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/expressions/class/class-field-extends-
```

Comment:
"expressions" => "statements" (for both files)

These templates create class declarations, which should end up in test/language/statements/class/